### PR TITLE
undoing live site brokenness

### PR DESCRIPTION
**Pull request by gvonkoss**
@TheoLeanse @wheresrhys

### CODE REVIEW

File: styles/nav-info.scss [hunk 1]
```diff
@@ -32,6 +32,6 @@
 }
 
 .n-storylines__back-btn {
-	border: transparent;
-	background: transparent;
+	border: transparent !important;
```
Comment (wheresrhys) [inline styles/nav-info.scss]: why these?

File: bower.json [hunk 2]
```diff
@@ -7,6 +7,7 @@
     "o-colors": "^3.5.4",
```
Comment (wheresrhys) [inline bower.json]: you can remove all these o-colors, o-icons etc dependencies too. The fewer components explictly declare dependency on them, the easier it is to roll out new versions

**Review by wheresrhys — CHANGES_REQUESTED** [review]

File: styles/nav-info.scss (hunk 1 above)
Comment (gvonkoss) [reply styles/nav-info.scss]: I was trying something, and forgot to remove that bit of styling!

**Comment by TheoLeanse** [comment]
I don't get why this build is failing...?

**Comment by ironsidevsquincy** [comment]
If you do a `bower install` locally, you'll see o-icons needs to be resolved. Resolve it to the latest version (v5.2.0)

**Comment by wheresrhys** [comment]
It'd be great to catch those errors and throw another more informative one

**Comment by onishiweb** [comment]
Have widened the version of `o-icons` allowed in `o-video` to fix the errors introduced by the `n-` components upgrade: https://github.com/Financial-Times/o-video/releases/tag/v2.3.17

**Comment by gvonkoss** [comment]
@onishiweb success!!

**Review by wheresrhys — APPROVED** [review]

**Pull request merged**